pennylane/devices/default_mixed.py

```diff
@@ -134,20 +146,33 @@ def _asarray(array, dtype=None):
         return res
 
     def __init__(
-        self, wires, *, r_dtype=np.float64, c_dtype=np.complex128, shots=None, analytic=None
+        self, wires, *, r_dtype=np.float64, c_dtype=np.complex128, shots=None, analytic=None, readout_prob=None
     ):
         if isinstance(wires, int) and wires > 23:
             raise ValueError(
                 "This device does not currently support computations on more than 23 wires"
             )
 
+        self.readout_err = readout_prob
+        # Check that the readout error probability, if entered, is either integer or float in [0,1]
+        if self.readout_err is not None:
+            if not isinstance(self.readout_err, float) and not isinstance(self.readout_err, int):
+                raise TypeError(
+                    "The readout error probability should be an integer or a floating-point number in [0,1]."
+                )
+            if self.readout_err < 0 or self.readout_err > 1:
+                raise ValueError(
+                    "The readout error probability should be in the range [0,1]."
+                )
+
         # call QubitDevice init
         super().__init__(wires, shots, r_dtype=r_dtype, c_dtype=c_dtype, analytic=analytic)
         self._debugger = None
 
         # Create the initial state.
         self._state = self._create_basis_state(0)
         self._pre_rotated_state = self._state
+        self.measured_wires = []
```

Comment:
At times, instance attributes can also have a small docstring. Here it may make sense to describe why we're keeping tracking of the measured wires. The provided description is just a suggestion.
```suggestion
        self.measured_wires = []
        """List: during execution, the wires on which measurements are acted on are obtained such that the readout error can be applied to them."""
```